docs/en/setup/service-agent/java-agent/agent-optional-plugins/Logger-plugin.md

```diff
@@ -0,0 +1,49 @@
+# Overview
```

Comment:
I think we miss a part about how to disable one log framework, such as opening the bridge for log4j only.

Reply:
I have removed `logconfig.properties` and add some use cases in the document.